.github/workflows/main.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       shell: powershell
     - name: Build Math libs & add to PATH
       run: |
+        make print-PRECOMPILED_HEADERS
```

Comment:
This is just a test that shows that its still disabled in the Github Action test with Rtools 3.5.